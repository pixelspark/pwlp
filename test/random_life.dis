0000.	e0	USER	get_length
0001.	20	PEEKB	0
0002.	11	PUSHB	[01]
0004.	81	BINARY	SUB
0005.	10	PUSHB	0
0006.	e3	USER	set_pixel
0007.	01	POP	1
0008.	71	UNARY	DEC
0009.	60	JNZ	to 1
0012.	01	POP	1
0013.	e0	USER	get_length
0014.	e5	USER	random_int
0015.	e0	USER	get_length
0016.	20	PEEKB	0
0017.	22	PEEKB	2
0018.	8c	BINARY	EQ
0019.	50	JZ	to 30
0022.	21	PEEKB	1
0023.	31	PUSHI	[ff, ff, ff, 00]
0028.	e3	USER	set_pixel
0029.	01	POP	1
0030.	01	POP	1
0031.	71	UNARY	DEC
0032.	60	JNZ	to 16
0035.	01	POP	1
0036.	e4	USER	blit
0037.	e0	USER	get_length
0038.	e5	USER	random_int
0039.	e0	USER	get_length
0040.	20	PEEKB	0
0041.	22	PEEKB	2
0042.	8c	BINARY	EQ
0043.	50	JZ	to 53
0046.	21	PEEKB	1
0047.	11	PUSHB	[01]
0049.	81	BINARY	SUB
0050.	10	PUSHB	0
0051.	e3	USER	set_pixel
0052.	01	POP	1
0053.	01	POP	1
0054.	71	UNARY	DEC
0055.	60	JNZ	to 40
0058.	01	POP	1
0059.	e4	USER	blit
0060.	02	POP	2
0061.	40	JMP	to 13

