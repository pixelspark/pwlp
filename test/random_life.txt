for(n=get_length) {
	set_pixel(n-1, 0, 0, 0);
};
loop {
	lucky = random(get_length);
	for(n=get_length) {
		if(n == lucky) {
			set_pixel(n, 255, 255, 255);
		}
	};
	blit;
	luckyb = random(get_length);
	for(n=get_length) {
		if(n == luckyb) {
			set_pixel(n-1, 0, 0, 0);
		}
	};
	blit;
}