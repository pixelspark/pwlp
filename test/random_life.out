000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff 000000 000000 000000 ffffff 000000 000000 000000 000000 
000000 000000 000000 000000 000000 ffffff 000000 000000 000000 000000 
000000 000000 ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 000000 ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 ffffff 000000 000000 000000 000000 
000000 ffffff ffffff 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 000000 000000 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 000000 000000 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 000000 ffffff 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 000000 ffffff 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 ffffff ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
000000 000000 ffffff 000000 ffffff 000000 000000 000000 ffffff ffffff 
